ai/vicuna/deploy/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.9"
```

Comment:
If Long or others participated in the development of the Dockerfile, please add his name in the form of a co-author to the commit message. You can refer to https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reply:
updated

Reply:
Please update commit sign-off info like this:
Signed-off-by: Guoyang Zhang <Your email>
Signed-off-by: Long Zhang <contenfire@qq.com>
Co-authored-by: Long Zhang <contenfire@qq.com>

Reply:
updated